mlir/test/Dialect/Rock/index_diff_update.mlir

```diff
@@ -27,6 +27,11 @@
         #rock.transform<PassThrough ["y"] at [1] -> ["y"] at [0]>]
     bounds = [16, 4] -> [4]>
 
+#transform_map6 = #rock.transform_map<affine_map<(d0) -> (d0, 1)>
+    by [<PassThrough ["x"] at [0] -> ["x"] at [0]>,
+        <ConstDim{1, 8} [] at [] -> ["y"] at [1]>]
```

Comment:
Does `ConstDim{1,8}` mean add a constant dimension at 1 with size 8?

Reply:
Nope. It means "add constant dimension with value 1 and length 8"

Reply:
Yes, that makes sense. Thanks!